signup.html

```diff
@@ -35,62 +27,109 @@ <h1>Signup Form</h1>
+            <button type="button" class="signupbtn" onclick="submitForm()">
+              Submit
+            </button>
           </div>
         </div>
       </form>
-      <script>
-        document.getElementById("myForm").onsubmit = function (event) {
-          let isValid = true;
+      <div class="response-message" id="responseMessage"></div>
+    </section>
+    <footer>
+      <div class="footer">
+        <a href="https://www.linkedin.com/in/fazila-abbasi-a07b18264/"
+          >Link to My LinkedIn Profile!</a
+        >
+      </div>
+    </footer>
 
-          // Reset error messages and field styles
-          document.getElementById("nameError").textContent = "";
-          document.getElementById("name").classList.remove("error-field");
-          document.getElementById("emailError").textContent = "";
-          document.getElementById("email").classList.remove("error-field");
-          document.getElementById("passwordError").textContent = "";
-          document.getElementById("password").classList.remove("error-field");
+    <script>
+      async function submitForm() {
+        let isValid = validateForm();
 
-          // Validation for Name
+        if (isValid) {
           const name = document.getElementById("name").value;
-          if (name.trim() === "") {
-            document.getElementById("nameError").textContent =
-              "Name is required.";
-            document.getElementById("name").classList.add("error-field");
-            isValid = false;
-          }
-
-          // Validation for Email
           const email = document.getElementById("email").value;
-          const emailPattern = /\S+@\S+\.\S+/;
-          if (!emailPattern.test(email)) {
-            document.getElementById("emailError").textContent =
-              "Invalid email address.";
-            document.getElementById("email").classList.add("error-field");
-            isValid = false;
-          }
-
-          // Validation for Password (length must be at least 8 characters)
           const password = document.getElementById("password").value;
-          if (password.length < 8) {
-            document.getElementById("passwordError").textContent =
-              "Password must be at least 8 characters.";
-            document.getElementById("password").classList.add("error-field");
-            isValid = false;
-          }
 
-          if (!isValid) {
-            event.preventDefault(); // Prevent form submission
+          try {
+            const response = await fetch("https://dummyjson.com/users/add", {
+              method: "POST",
+              headers: { "Content-Type": "application/json" },
+              body: JSON.stringify({
+                name: name,
+                email: email,
+                password: password,
+              }),
+            });
+
+            const result = await response.json();
+
+            console.log(name, email, password); // Log the user details to the console
```

Comment:
Please avoid submitting code with console logs.

Reply:
Please check in other places as well

Reply:
fix